src/app/[locale]/_components/DashboardInput.tsx

```diff
@@ -1,50 +1,43 @@
 'use client';
-import { Box, Typography, useTheme } from '@mui/material';
-import ScannerLogo from 'assets/ScannerLogo.svg';
-import { useIsMobile } from 'lib/hooks/UseBreakpoints';
+import { Typography } from '@mui/material';
 import { messages } from 'lib/i18n/localization';
-import { useState } from 'react';
 import { FormattedMessage } from 'react-intl';
-import { ManualAASViewerInput } from '../viewer/_components/ManualAasViewerInput';
+import { ManualAasInput } from 'app/[locale]/viewer/_components/ManualAasInput';
```

Comment:
I would use `../viewer/_components/ManualAasViewerInput`

Reply:
I disagree tbh. We didn't rellay define it anywhere, but up until now we mostly used the absolute imports